graph/mutators/comment.js

```diff
@@ -187,11 +187,33 @@ const setCommentStatus = ({loaders: {Comments}}, {id, status}) => {
     });
 };
 
+/**
+ * Adds a tag to a Comment
+ * @param {String} id          identifier of the comment  (uuid)
+ * @param {String} tag     name of the tag
+ */
+const addCommentTag = ({user, loaders: {Comments}}, {id, tag}) => {
+  return CommentsService.addTag(id, tag, user.id)
+    .then(() => CommentsService.findById( id ));
```

Comment:
@wyattjoh Alright. I did that originally, then put this here as the quickest way of Apollo being able to update the Comments cache to reflect the changed tag.

How do you recommend I have the client update its representation of the Comment? Originally I looked to see if there was a single comment query I could make and use with refetchQueries, but I didn't see one. And it seems heavy to totally refetch the whole comment stream of the asset.

Should I use [updateQueries](http://dev.apollodata.com/react/cache-updates.html#updateQueries) and manually update streamQuery, loadMore, and myCommentHistory queries that fetch comments?

Is there a way to update the cached comment directly?